client/src/App.js

```diff
@@ -12,23 +13,32 @@ import BottomNav from "./components/BottomNav";
 const queryClient = new QueryClient();
 
 function App() {
+  const [selectedImage, setSelectedImage] = useState(null);
   return (
     <div className="App">
       <QueryClientProvider client={queryClient}>
-        <Page />
         <Router>
-          <Switch>
-            <Route exact path="/">
-              <UploadPage />
-            </Route>
-            <Route exact path="/tagging">
-              <TaggingPage />
-            </Route>
-            <Route exact path="/gallery">
-              <GalleryPage />
-            </Route>
-          </Switch>
-          <BottomNav />
+          <PageContainer>
+            <TopElement />
+            <Switch>
+              <Route exact path="/">
+                <UploadPage />
+              </Route>
+              <Route exact path="/tagging">
```

Comment:
exact path should only be necessary for the "home"-Link ("/"), so here you can remove "exact"